package.json

```diff
@@ -3,6 +3,8 @@
     "@openzeppelin/contracts": "^3.3.0"
   },
   "devDependencies": {
+    "@keep-network/keep-core": ">1.8.0-dev <1.8.0-pre",
+    "@keep-network/common.js": "0.0.1-3",
```

Comment:
Can we avoid creating this dependency? KEEP Token Dashboard should depend on coverage pools but there should be no dependency between core contracts and coverage pool contracts.

Reply:
We need the KEEP token address as a `collateralToken`, right? This dependency is needed to fetch KEEP token address. We do the same in the [`keep-ecdsa`](https://github.com/keep-network/keep-ecdsa/blob/master/solidity/migrations/2_deploy_contracts.js#L39-L63) to fetch eg. `Random Beacon` address.